src/consts.rs

```diff
@@ -90,12 +90,12 @@ impl Constant {
 impl PartialEq for Constant {
     fn eq(&self, other: &Constant) -> bool {
         match (self, other) {
-            (&Constant::Str(ref ls, ref lsty), &Constant::Str(ref rs, ref rsty)) => ls == rs && lsty == rsty,
+            (&Constant::Str(ref ls, ref l_sty), &Constant::Str(ref rs, ref r_sty)) => ls == rs && l_sty == r_sty,
```

Comment:
xD i thought so, too. But once I added the underscore, I found it much better. If it were a type, you wouldn't name it `Rsty` but you'd name it `RSty`. This is the snake-case variant of that.